README.md

```diff
@@ -30,7 +30,7 @@ packages:
 ```
 
 ## Package Maintenance
-The Fivetran team maintaining this package **only** maintains the latest version. We highly recommend you keep your `packages.yml` updated with the [dbt hub latest version](https://hub.getdbt.com/fivetran/social_media_reporting/latest/). You may refer to the [CHANGELOG](/CHANGELOG.md) and release notes for more information on changes across versions.
+The Fivetran team maintaining this package **only** maintains the latest version. We highly recommend that you keep your `packages.yml` file updated with the [dbt hub's latest version](https://hub.getdbt.com/fivetran/social_media_reporting/latest/). Read the [CHANGELOG](/CHANGELOG.md) and release notes for more information on changes across versions.
```

Comment:
We mean the latest version in the dbt hub, correct?

Reply:
That is correct. Should we explicitly identify the dbt hub here?

Reply:
I may just reword it to make that clear

Reply:
Ok, made the edit - what do you think?

Reply:
That is great, thanks!